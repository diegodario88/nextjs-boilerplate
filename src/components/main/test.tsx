import { screen, render } from '@testing-library/react';
import Main from './index';

describe('<Main />', () => {
  it('Should Render heading', () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole('heading', { name: /Testing with Jest/ })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
