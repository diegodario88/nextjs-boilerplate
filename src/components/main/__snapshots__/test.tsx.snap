// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should Render heading 1`] = `
<main
  class="styles__Wrapper-cwsudp-0 kABIWL"
>
  <img
    alt="Imagem do mapa mÃºndi"
    class="styles__Logo-cwsudp-1 gGOyxR"
    src="/images/world-map.svg"
  />
  <h1
    class="styles__Title-cwsudp-2 jnUjSy"
  >
    Testing with Jest
  </h1>
  <h2
    class="styles__Description-cwsudp-3 ctHSGV"
  >
    Typescript, ReactJS e Styled Components
  </h2>
  <img
    alt="Desenvolvedor atuando em um computador"
    class="styles__Illustration-cwsudp-4 kLdOcw"
    src="/images/hero-illustration.svg"
  />
</main>
`;
